src/chocolatey/infrastructure.app/commands/ChocolateyHelpCommand.cs

```diff
@@ -0,0 +1,303 @@
+﻿// Copyright © 2017 - 2018 Chocolatey Software, Inc
+// Copyright © 2011 - 2017 RealDimensions Software, LLC
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// 
+// You may obtain a copy of the License at
+// 
+// 	http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+namespace chocolatey.infrastructure.app.commands
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+    using System.Text;
+    using attributes;
+    using commandline;
+    using configuration;
+    using infrastructure.commands;
+    using logging;
+    using results;
+    using services;
+    using SimpleInjector;
+
+
+    [CommandFor("help", "List some help stuffs...")]
+    public class ChocolateyHelpCommand : IListCommand<PackageResult>
+    {
+        private readonly IChocolateyPackageService _packageService;
+
+        public ChocolateyHelpCommand(IChocolateyPackageService packageService)
+        {
+            _packageService = packageService;

```

Comment:
I missed this - you won't need a package service (unless I missed it and it is used). Remove the readonly variable _packageService, remove this, and the constructor as it won't need it (since empty defaults don't need to be created).